src/lib/openapi/spec/strategy-schema.ts

```diff
@@ -13,6 +13,9 @@ export const strategySchema = {
         'parameters',
     ],
     properties: {
+        title: {
```

Comment:
I'd add descriptions and examples in this or next PR

Reply:
Good call, I will fix the schema since I am here

Reply:
Good call.